### ISSUE TITLE
No Activity

### ISSUE THREAD

**Issue by abeputnam** [pauleeeeee/just-chat#4]
I love this concept, but there's never anyone else using the app at the same time as me. Would there be a way to run the app in the background and notify you when somebody adds a message?

**Comment by abeputnam** [comment]
There are obviously many ways the engagement problem could be fixed, such as adding a set time people get on and chat under the app store description, just to boost actual conversation while more features are implemented.

**Comment by pauleeeeee** [comment]
Yes I agree - do you know JavaScript? I can convert it to a PebbleJS project and you can try to enhance it

**Comment by abrahamputnam** [comment]
I’m learning on Khan Academy. 😅

**Comment by abrahamputnam** [comment]
Would I have to install the SDK?

**Comment by abeputnam** [comment]
> Yes I agree - do you know JavaScript? I can convert it to a PebbleJS project and you can try to enhance it

I'm learning on Khan Academy 😅 would I install the SDK etc.?

**Comment by pauleeeeee** [comment]
So I tried to convert it to PebbleJS but I haven't finished that yet. Yes you would need to install the SDK. If you would like help doing this stuff you can come to the Rebble discord https://discord.com/invite/aRUAYFN
